src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -125,7 +128,7 @@ public function collect(Request $request, Response $response, \Throwable $except
             }
 
             foreach ($this->data as $dump) {
-                $this->doDump($dumper, $dump['data'], $dump['name'], $dump['file'], $dump['line']);
+                $this->doDump($dumper, $dump['data'], $dump['name'], $dump['file'], $dump['line'], $dump['label'] ?? '');
```

Comment:
Doesn't the label key always exist (same below)?

Reply:
I thought about existing profiles, which we might load from a previous version during migrations.